packages/cli/src/generators/index.js

```diff
@@ -130,17 +130,15 @@ const writeForStandaloneTemplate = (gen) => {
     },
     typescript: {
       scripts: {
+        test: 'vitest',
```

Comment:
Vitest will enter watch mode by default for users in interactive sessions. If the `CI` environment variable is set or it detects that its output is being piped or redirected (such as when run as part of a script), it will run once and then exit.

This behaviour can be forced with `vitest --run`, but watch mode is a nice quality of life improvement as a default.